tests/test.h

```diff
@@ -150,4 +150,12 @@ void same_param_value(char* dev_name, uint64_t uid, char* param_name, param_type
  */
 void check_param_range(char* dev_name, uint64_t uid, char* param_name, param_type_t param_type, param_val_t expected_low, param_val_t expected_high);
 
+/**
+* Checks the latency between an action and the TIMESTAMP parameter on a TimeTestDevice
+ * Arguments:
+ *    uid: unique identifier of TimeTestDevice
+ *    upper_bound_latency: The expected upperbound latency
+ *    start_time: The start of a timer provided by the test case(in ms)
+*/
+void check_latency(uint64_t uid, int32_t upper_bound_latency, uint32_t start_time);
```

Comment:
Hm so in the test case, you pass in a call to `millis()` as the start time, where `millis()` returns a `uint64_t`.

You can make the parameter type `uint64_t` as well, and mention in the function comment that `start_time` should be some call to `millis()`

Reply:
Oh yea I forgot to have the `.h` reflect that change. Thanks for the catch